### ISSUE TITLE
Trouble installing Python from Windows Store and App Execution Aliases

### ISSUE THREAD

**Issue by TomDoingArt** [MicrosoftDocs/windows-uwp#2118]
I had trouble with the Python app install from Microsoft Store. There is a comment in the reviews on the store for Python 3.8 which helped me. I had the same error trying to install 3.7. I had to: "open 'Manage App Execution Aliases' through Start and disable all the Python entries before installing."

App Execution Aliases came back to bother me again when I tried to run python from the shell. It started up the Windows Store page. I moved the folder for these aliases further down in my PATH environment variable. That made it so that an installed instance of python came first before those redirects.

Finally after learning more about what App. Ex. Aliases are...I cleaned up the settings on the Manage App Execution Aliases page in Control Panel. It allows me to default which version of Python is my default when I type "python" from a command prompt.

I found this helpful:
https://docs.microsoft.com/en-us/windows/python/faqs#why-does-running-pythonexe-open-the-microsoft-store

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e6092806-2037-a886-8895-12012f37ef7e
* Version Independent ID: 8b5aff2a-668b-f79d-ba29-e5348a09a0b5
* Content: [Get started with using Python on Windows for beginners](https://docs.microsoft.com/en-us/windows/python/beginners)
* Content Source: [hub/python/beginners.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/hub/python/beginners.md)
* Product: **dev-environment**
* Technology: **windows-python**
* GitHub Login: @mattwojo
* Microsoft Alias: **mattwoj**

**Comment by mattwojo** [comment]
Thanks for posting this @zentril! I just got in touch with our Python team and here's the reply that I received: "A fix has been merged (for the regression in a security patch) and hopefully it will go out very soon – this is affecting 75-80% of Python installs from the Store right now."

**Comment by mattwojo** [comment]
I'll be paying attention and post an update once I see one.

**Comment by zooba** [comment]
Hi @zentril 

I have some follow-up questions about this statement:
> App Execution Aliases came back to bother me again when I tried to run python from the shell. It started up the Windows Store page. I moved the folder for these aliases further down in my PATH environment variable. That made it so that an installed instance of python came first before those redirects.

When you tried to run "python from the shell", was it the Store install you were looking for? Or another install that came from python.org?

Also, when you moved your aliases folder down in your PATH, did you happen to notice:
* whether it started with `C:\Users\<your name>` or `%USERPROFILE%`?
* whether there was more than one entry in PATH going to `WindowsApps`?

And finally, do you have Visual Studio 2017/2019 installed? And if you remember, did you install it _before_ your last Windows Update was installed?

Thanks! These have been a difficult 4-5 bugs to track down but we're getting close, hence my oddly specific questions.

**Comment by TomDoingArt** [comment]
Hi @zooba 

It was the store install python I was looking for, 3.8, when I tried to run the python command in the shell. I had no prior install of Python so I did notice when trying to run it from the cmd prompt yesterday it did take me to the store. That was before I found and started to follow your tutorial.

I did have two entries pointing to the same place, one started with %USERPROFILE% and it was the higher of the two. The other had my user name explicitly and I believe it was already lower on the path than the Microsoft VS Code\bin entry. The WindowsApps entry with my user name explicit is the last item on my user environment PATH variable now. I deleted the entry with %USERPROFILE% after confirming they both pointed to the same place.

I do have Visual Studio 2017 installed and have had it installed for a while so definitely before the last windows update.

**Comment by TomDoingArt** [comment]
> I'll be paying attention and post an update once I see one.

Thank you @mattwojo

**Comment by zooba** [comment]
Thanks @zentril. Glad you got everything sorted out, and sorry you ran into these issues in the first place.

**Comment by TomDoingArt** [comment]
> Thanks @zentril. Glad you got everything sorted out, and sorry you ran into these issues in the first place.

Thank you. It was a little discouraging being stopped at the first step but was happy to get past it. Saw the other issue someone reported regarding the additional python component for VS Code. I got a message about that during the tutorial so knowing about it I followed some prompts and got that installed. That motivated me to put in my own comment / issue report here to help strengthen your great intro tutorial.
